assets/blocks/course-list-block/course-list.scss

```diff
@@ -26,10 +26,15 @@ $block: '.wp-block-sensei-lms-course-list';
 		/* Align button to bottom of card. */
 		.wp-block-post {
 			display: flex;
```

Comment:
I think we can remove this `display: flex` since we added it on the Group block now.